packages/flutter_tools/test/general.shard/android/gradle_test.dart

```diff
@@ -1973,6 +1976,7 @@ plugin2=${plugin2.path}
         buildModes: const <String>{'profile'},
         androidPackage: 'com.mycompany',
         repoDirectory: fs.directory('build/'),
+        buildNumber: '1.0',
       );
 
       expect(
```

Comment:
It also needs to update the call to Gradle in `buildPluginsAsAar -> calls gradle`